tasks/go.py

```diff
@@ -438,6 +440,57 @@ def is_excluded(pkg):
     return licenses
 
 
+@task
+def generate_protobuf(ctx):
+    """
+    Generates protobuf defintions in pkg/proto
+    """
+    base = os.path.dirname(os.path.abspath(__file__))
+    repo_root = os.path.abspath(os.path.join(base, ".."))
+    proto_root = os.path.join(repo_root, "pkg", "proto")
+
+    print("nuking old definitions at: {}".format(proto_root))
+    file_list = glob.glob(os.path.join(proto_root, "pbgo", "*.go"))
+    for file_path in file_list:
+        try:
+            os.remove(file_path)
+        except OSError:
+            print("Error while deleting file : ", file_path)
+
+    with ctx.cd(repo_root):
+        # protobuf defs
+        print("generating protobuf code from: {}".format(proto_root))
+
+        files = []
+        for path in Path(os.path.join(proto_root, "datadog")).rglob('*.proto'):
+            files.append(path.as_posix())
+
+        ctx.run(
+            "protoc -I{include_path} --go_out=plugins=grpc:{out_path} {targets}".format(
```

Comment:
Not something to be fixed now but we really need to allow `ctx run` to take in an array of strings instead of just a string.

Reply:
I assume this is using whatever local version of `protoc` that is installed. Does the version of `protoc` installed matter or effect proto generation?

Reply:
See: 
- https://github.com/DataDog/datadog-agent/tree/jaime/protobuf_refactor/internal/tools/proto
- https://github.com/DataDog/datadog-agent/blob/c5ce993c5ec133f57584302d2d35009477826bfa/tasks/test.py#L63-L67